sql_user_check_mongodb/run.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+readonly connection_string=$connection_string
+
+if ! command -v "mongo" &> /dev/null
+then
+    echo "Command mongo could not be found, Follow https://stackoverflow.com/questions/58504865/install-mongo-shell-in-mac to install mongo"
+    exit 1
+fi
```

Comment:
Isn't the command supposed to be mongosh?

Also let's not give Stackoverflow links. Let's provide official docs link or no link

Reply:
done, mongo was working too.